torch/csrc/dynamo/guards.cpp

```diff
@@ -0,0 +1,378 @@
+#define PY_SSIZE_T_CLEAN
```

Comment:
This file is mostly copied from https://github.com/pytorch/torchdynamo/blob/main/torchdynamo/_guards.cpp